app/api/probe_dock/users_api.rb

```diff
@@ -21,23 +21,25 @@ class UsersApi < Grape::API
     namespace :users do
       helpers do
         def parse_user
-          parse_object :name, :primaryEmail, :active, :technical, :organizationId, :password, :passwordConfirmation
+          parse_object(:name, :primaryEmail, :active, :technical, :organizationId, :password, :passwordConfirmation)
         end
 
         def current_otp_record
-          @current_otp_record ||= if params.key? :membershipOtp
+          @current_otp_record ||= if params.key?(:membershipOtp)
             Membership.where('otp IS NOT NULL').where(otp: params[:membershipOtp].to_s).where('expires_at > ?', Time.now).first
           end
         end
 
         def with_serialization_includes rel
-          rel = rel.includes :memberships if true_flag? :withTechnicalMembership
+          rel = rel.includes(:memberships) if true_flag?(:withTechnicalMembership)
+          rel = rel.includes(:organizations) if true_flag?(:withOrganizations)
```

Comment:
Also include e-mails, otherwise:

```
  User Load (0.5ms)  SELECT  "users".* FROM "users"  ORDER BY LOWER(users.name) ASC LIMIT 15 OFFSET 0
  Membership Load (0.5ms)  SELECT "memberships".* FROM "memberships" WHERE "memberships"."user_id" IN (20, 1, 10, 30, 16, 6, 17, 5, 24, 13, 9, 25, 18, 8, 11)
  Organization Load (0.4ms)  SELECT "organizations".* FROM "organizations" WHERE "organizations"."id" IN (10, 1, 5, 11, 9, 4)
  Email Load (0.3ms)  SELECT  "emails".* FROM "emails" WHERE "emails"."id" = $1 LIMIT 1  [["id", 25]]
  Email Load (0.2ms)  SELECT "emails".* FROM "emails" WHERE "emails"."user_id" = $1  [["user_id", 20]]
  Email Load (0.2ms)  SELECT  "emails".* FROM "emails" WHERE "emails"."id" = $1 LIMIT 1  [["id", 1]]
  Email Load (0.1ms)  SELECT "emails".* FROM "emails" WHERE "emails"."user_id" = $1  [["user_id", 1]]
  Email Load (0.2ms)  SELECT  "emails".* FROM "emails" WHERE "emails"."id" = $1 LIMIT 1  [["id", 12]]
  Email Load (0.1ms)  SELECT "emails".* FROM "emails" WHERE "emails"."user_id" = $1  [["user_id", 10]]
  Email Load (0.2ms)  SELECT  "emails".* FROM "emails" WHERE "emails"."id" = $1 LIMIT 1  [["id", 35]]
  Email Load (0.1ms)  SELECT "emails".* FROM "emails" WHERE "emails"."user_id" = $1  [["user_id", 30]]
  Email Load (0.2ms)  SELECT  "emails".* FROM "emails" WHERE "emails"."id" = $1 LIMIT 1  [["id", 20]]
  Email Load (0.1ms)  SELECT "emails".* FROM "emails" WHERE "emails"."user_id" = $1  [["user_id", 16]]
  Email Load (0.2ms)  SELECT  "emails".* FROM "emails" WHERE "emails"."id" = $1 LIMIT 1  [["id", 6]]
  Email Load (0.2ms)  SELECT "emails".* FROM "emails" WHERE "emails"."user_id" = $1  [["user_id", 6]]
  Email Load (0.3ms)  SELECT  "emails".* FROM "emails" WHERE "emails"."id" = $1 LIMIT 1  [["id", 21]]
  Email Load (0.2ms)  SELECT "emails".* FROM "emails" WHERE "emails"."user_id" = $1  [["user_id", 17]]
  Email Load (0.3ms)  SELECT  "emails".* FROM "emails" WHERE "emails"."id" = $1 LIMIT 1  [["id", 5]]
  Email Load (0.2ms)  SELECT "emails".* FROM "emails" WHERE "emails"."user_id" = $1  [["user_id", 5]]
  Email Load (0.3ms)  SELECT  "emails".* FROM "emails" WHERE "emails"."id" = $1 LIMIT 1  [["id", 29]]
  Email Load (0.2ms)  SELECT "emails".* FROM "emails" WHERE "emails"."user_id" = $1  [["user_id", 24]]
  Email Load (0.2ms)  SELECT  "emails".* FROM "emails" WHERE "emails"."id" = $1 LIMIT 1  [["id", 22]]
  Email Load (0.2ms)  SELECT "emails".* FROM "emails" WHERE "emails"."user_id" = $1  [["user_id", 18]]
  Email Load (0.2ms)  SELECT  "emails".* FROM "emails" WHERE "emails"."id" = $1 LIMIT 1  [["id", 11]]
  Email Load (0.2ms)  SELECT "emails".* FROM "emails" WHERE "emails"."user_id" = $1  [["user_id", 8]]
  Email Load (0.2ms)  SELECT  "emails".* FROM "emails" WHERE "emails"."id" = $1 LIMIT 1  [["id", 14]]
  Email Load (0.2ms)  SELECT "emails".* FROM "emails" WHERE "emails"."user_id" = $1  [["user_id", 11]]
```

Since the conditions of when e-mails are loaded are [quite complex](https://github.com/probedock/probedock/blob/release-8/app/policies/user_policy.rb#L90-L94), I would advise to simply include them here anyway without checking for any condition.